shoov/modules/custom/shoov_ci_build/shoov_ci_build.module

```diff
@@ -174,31 +182,100 @@ function shoov_ci_build_message_update(Message $message) {
 
   $node = $wrapper->field_ci_build->value();
 
+  shoov_ci_build_process_ci_build_item($message);
+}
+
+/**
+ * Return the timestamp for the next CI build item execution.
+ *
+ * @param Message $message
+ *  The CI build item.
+ * @return int|null
+ *  The time of start next CI build item.
+ *
+ */
+function shoov_ci_build_get_timestamp_for_next_ci_build_item(Message $message) {
+  $wrapper = entity_metadata_wrapper('message', $message);
+  $node = $wrapper->field_ci_build->value();
+
+  if (shoov_ci_build_get_ci_build_status($node) == SHOOV_CI_BUILD_CI_BUILD_STATUS_UNCONFIRMED_ERROR) {
```

Comment:
@amitaibu  Problem it is that on this moment ci build have OK status. Because ci build item not yet process. Therefore new item get 3 minutes instead of 1 if error happend.